Drivers/Inc/CANConfig.h

```diff
@@ -15,7 +15,7 @@
  * Filter Lists for CarCAN and MotorCAN
 */
 #define NUM_CARCAN_FILTERS 6
-#define NUM_MOTORCAN_FILTERS 0
+#define NUM_MOTORCAN_FILTERS 7
```

Comment:
im guessing this is just the number of IDs to allow in right?

Reply:
yup. honestly we could use sizeof for the array instead of this macro.

Reply:
I think we can merge it as is and address it later. #387